packages/devtools_app/lib/src/timeline/flutter/flutter_frames_chart.dart

```diff
@@ -61,6 +61,32 @@ class _FlutterFramesChartState extends State<FlutterFramesChart>
 
   final int totalFramesToChart = 150;
 
+  /// Compute the FPS highwater mark, 60 FPS is 16.6 ms and 120 FPS is 8 ms.
+  void _setupFPSHighwaterLine() async {
+    if (_chartController.axisLeftSettingFunction == null) {
+      final fpsRate =
+          (await _controller.frameBasedTimeline.displayRefreshRate).toInt();
+      final fpsInMs = fpsRate == 60 ? 16.6 : 8.0;
+
```

Comment:
Leave fpsRate as a double and then rename fpsInMs to targetMsPerFrame or similar. Then 

`final targetMsPerFrame = 1/fpsRate * 1000;`

Reply:
Used your computation, kept as double and changed output of FPS label to not display radix and decimal part of number.